src/services/Clips.ts

```diff
@@ -0,0 +1,76 @@
+import { IClipProperties } from '../defs/clipProperties';
+import { IClipRequest } from '../defs/clipRequest';
+import { IResponse } from '../RequestRunner';
+import { Service } from './Service';
+
+/**
+ * Service for interacting with clips on the Mixer REST API.
+ */
+export class ClipsService extends Service {
+
+    /**
+     * Can a clip be created on the given broadcast.
+     */
+    public async canClip(broadcastId: string): Promise<boolean> {
+        try {
+            await this.makeHandled<boolean>('get', `clips/broadcasts/${broadcastId}/canClip`);
+        } catch {
+            return Promise.resolve(false);
+        }
+
+        return Promise.resolve(true);
+    }
+
+    /**
+     * Creates a clip.
+     */
+    public async createClip(p: IClipRequest): Promise<boolean> {
```

Comment:
Please avoid single char variable names.

Reply:
Opps. Meant to change this